src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -37,8 +37,8 @@ const RichTextEditor: React.FC<RichTextEditorProps> = ({
 
   React.useEffect(
     () => {
-      if (data !== undefined) {
-        editor.current = new EditorJS({
+      if (data !== undefined && !editor.current) {
```

Comment:
We really need to get the source of the issues instead of playing whac-a-mole, it looks like multiple workarounds are already laid on top of each other. It's fine if we merge it in if this gets tested extensively but I'd recommend starting from scratch and taking a look at it from a bigger distance.